src/containers/Error/index.js

```diff
@@ -1,21 +1,25 @@
 import React, { PropTypes } from 'react';
 import IndexHeader from 'components/IndexHeader';
 import Helmet from 'react-helmet';
+import { FormattedHTMLMessage } from 'react-intl';
 
 const error = {
-  'invalid-surah': 'Surah is out of range',
-  'invalid-ayah-range': 'Ayah(s) selected are out of range'
+  'invalid-surah': "Surah is out of range. Please go to <a href='/'> home page</a> and select a Surah",
```

Comment:
Not sure if that will work since it's text and not JSX. Need to change this to be react code for that to work